backend.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  backend "s3" {
+    bucket         = "cisa-cool-terraform-state"
+    dynamodb_table = "terraform-state-lock"
+    encrypt        = true
+    key            = "cool-was-db-iam/terraform.tfstate"
+    profile        = "cool-terraform-backend"
+    region         = "us-east-1"
```

Comment:
Alphabetize.

```suggestion
    bucket         = "cisa-cool-terraform-state"
    dynamodb_table = "terraform-state-lock"
    encrypt        = true
    key            = "cool-was-db-iam/terraform.tfstate"
    profile        = "cool-terraform-backend"
    region         = "us-east-1"
```

Reply:
I'm going to resolve this conversation, since the suggestion is the same as what is already there. 😄  Please re-open if I'm missing something here.